Resources/doc/index.rst

```diff
@@ -64,8 +64,8 @@ Usage
 
 .. caution::
 
-    If your application is based on Symfony 3, replace ``php app/console`` by
-    ``php bin/console`` before executing any of the console commands included
+    If your application is based on Symfony <2.5.*, replace ``php bin/console`` by
```

Comment:
@stof The directory structure of Symfony was changed from version 2.5 (google search), that is why I chose 2.5.*. 

I have only worked in 2.1, 3.0, 3.1, so I am not sure if 2.6-8 used app/console or bin/console.